src/OrleansTestingHost/DefaultSiloBuilderFactory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Orleans.Hosting;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+
+namespace Orleans.TestingHost
+{
+    public class DefaultSiloBuilderFactory : ISiloBuilderFactory
+    {
+        public ISiloBuilder CreateSiloBuilder(string siloName, ClusterConfiguration clusterConfiguration)
+        {
+            ISiloBuilder builder = new SiloBuilder();
+            return builder.ConfigureSiloName(siloName).UseConfiguration(clusterConfiguration);
+        }
```

Comment:
you could even also provide a protected virtual method called ConfigureServices, so that derived classes don't need to override the main CreateSiloBuilder just to call their own ConfigureServices, as 99% of the times, that's what they'll do